l10n_es_aeat_mod303_cash_basis/models/account_move.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2018 Comunitea - Santi Argüeso
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    move_type_recc = fields.Selection(
+        string="Move type RECC", selection=[
+            ('other', 'Other'),
+            ('receivable', 'Receivable'),
+            ('receivable_refund', 'Receivable refund'),
+            ('payable', 'Payable'),
+            ('payable_refund', 'Payable refund'),
+        ], readonly=True)
+
+    def _compute_move_type_recc(self):
+        for move in self:
+            if move.tax_cash_basis_rec_id:
+                invoice_debit \
+                    = move.tax_cash_basis_rec_id.debit_move_id.invoice_id
+                invoice_credit \
+                    = move.tax_cash_basis_rec_id.credit_move_id.invoice_id
+                if invoice_debit:
+                    move.move_type_recc = invoice_debit.move_id.move_type
+                elif invoice_credit:
+                    move.move_type_recc = invoice_credit.move_id.move_type
+                else:
+                    move.move_type_recc = 'other'
+            else:
+                move.move_type_recc = 'other'
+
+    @api.multi
+    def reverse_moves(self, date=None, journal_id=None):
+        reversed_ids = super(AccountMove, self).reverse_moves(date, journal_id)
+        reversed = self.browse(reversed_ids)
+        reverse_types = {
+            'receivable': 'receivable_refund',
+            'receivable_refund': 'receivable',
+            'payable': 'payable_refund',
+            'payable_refund': 'payable',
+        }
+        for move in reversed:
+            if move.move_type_recc in ('receivable, payable, '
```

Comment:
`if move.move_type_recc in reverse_types.keys():`

Reply:
ok